### DISCUSSION TITLE
Organizing personal blog content

### DISCUSSION THREAD

**Discussion by fahmiidris** [markdoc/markdoc#76] [Q&A] [answered]
markdoc allows us to write markdown in the page folder. and I noticed that almost every post also has associated assets, like post banners, OG images, etc. Having to store them in another folder felt a bit sloppy, so I decided to give each post its own folder in the pages directory, and put the post content in an index.md file.

```
public/
src/
├── components/
├── css/
├── img/
└── pages/
    ├── post-2/
    │   ├── index.md
    │   └── card.jpeg
    ├── post-1/
    │   ├── index.md
    │   ├── css.png
    │   ├── html.png
    │   └── card.jpeg
    ├── _app.js
    ├── _document.js
    └── index.js
next.config.js
package.json
postcss.config.js
README.md
tailwind.config.js
```

can markdoc keep those assets in the same folder as the markdown content itself?

**Comment by mfix-stripe** [comment, marked as answer]
Markdoc isn't particular about how you store your assets. 

The Next.js API forces you to put your content in the `pages/` directory. You can control which files are turned into pages using [`pageExtensions`](https://nextjs.org/docs/api-reference/next.config.js/custom-page-extensions), which allows you to put other assets next to your page files.
